server/datastore/mysql/migrations/tables/20250311132525_AddAuthorIdToLabels.go

```diff
@@ -0,0 +1,31 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20250311132525, Down_20250311132525)
+}
+
+func Up_20250311132525(tx *sql.Tx) error {
+	_, err := tx.Exec(
+		"ALTER TABLE `labels` " +
```

Comment:
Similar to `author_id` in the `policies` and `queries` tables, we need a foreign constraint and `ON DELETE SET NULL`:
```
  CONSTRAINT `labels_users_fk_1` FOREIGN KEY (`author_id`) REFERENCES `users` (`id`) ON DELETE SET NULL
```

Reply:
Done 👍